functional-tests/src/test/java/com/redhat/che/functional/tests/Che7DialogAbout.java

```diff
@@ -108,25 +110,25 @@ public void openDialogAbout() {
   }
 
   private TestWorkspace createChe7Workspace() {
-    prepareWorkspace("che7", WORKSPACE_NAME, null);
+    prepareWorkspace(JAVA_MAVEN_DEVFILE_ID, WORKSPACE_NAME, null);
     projectSourcePage.clickOnAddOrImportProjectButton();
     newWorkspace.clickOnCreateButtonAndOpenInIDE();
     return testWorkspaceProvider.getWorkspace(WORKSPACE_NAME, defaultTestUser);
   }
 
-  private void prepareWorkspace(String stack, String workspaceName, Double machineRam) {
+  private void prepareWorkspace(String stackID, String workspaceName, Double machineRam) {
     dashboard.waitDashboardToolbarTitle();
     LOG.info("Opening Workspaces from left menu...");
     dashboard.selectWorkspacesItemOnDashboard();
     workspaces.clickOnAddWorkspaceBtn();
     newWorkspace.waitToolbar();
     LOG.info("Selecting stack...");
-    selectStack();
+    selectDevfile(stackID);
```

Comment:
@ScrewTSW why the argument is still `stackID` if in reality it is Devfile name ?

Reply:
refactored

Reply:
fixed :D also it's good that we get a few builds running, we are seeing some issues with deployments and tests getting stuck (usually problem with theia server not starting)